spark-launch/src/main/scala/com/ibm/sparktc/sparkbench/sparklaunch/SparkLaunch.scala

```diff
@@ -0,0 +1,63 @@
+package com.ibm.sparktc.sparkbench.sparklaunch
+
+import com.ibm.sparktc.sparkbench.utils.GeneralFunctions.getOrThrow
+import com.typesafe.config._
+import java.io.File
+import scala.collection.parallel.ForkJoinTaskSupport
+import scala.collection.JavaConverters._
+import scala.sys.process._
+import scala.util.Try
+
+object SparkLaunch extends App {
+
+  override def main(args: Array[String]): Unit = {
+    println("lol")
+    println(s"these are my argz: ${args.mkString(", ")}")
```

Comment:
Should these debug `println`s be removed?  Maybe we should make a ticket to convert debug prints like line 50 below to use Log4J.